src/drive/actions/async.js

```diff
@@ -2,8 +2,16 @@
 import { isCordova } from '../mobile/lib/device'
 import { TRASH_DIR_ID, FILES_FETCH_LIMIT } from '../constants/config.js'
 
+export const shouldShowRecentsFirst = (
+  folderPath,
+  parentPath,
+  specialFolders
+) =>
+  specialFolders.indexOf(folderPath) !== -1 ||
+  specialFolders.indexOf(parentPath) !== -1
```

Comment:
I wonder if it would be better to use regular expressions here, as it would avoid matching things like `/lol/Administrative`. Just a detail, but what do you think?

Reply:
It's not necessary imo, this condition doesn't match `/lol/Administrative`, only `/Administrative` and sub-folders.